.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,63 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          # New to Python?
+
+          The issue tracker isn't the right place to get help.
+          Consider instead:
+
+            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+            - posting on [Discourse](https://discuss.python.org/c/users/7)
+            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
```

"Consider instead..." renders on new line in the issue template -- not a minor issue, but I believe changing to the ``>`` form of multi-line would fix it.

```suggestion
      value: >
          # New to Python?

          The issue tracker isn't the right place to get help.
          Consider instead:

            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)

            - posting on [Discourse](https://discuss.python.org/c/users/7)

            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
```

Or, just collapsing onto the previous line:

```suggestion
      value: |
          # New to Python?

          The issue tracker isn't the right place to get help. Consider instead:

            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)
            - posting on [Discourse](https://discuss.python.org/c/users/7)
            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
```